### PR TITLE
D&a note

### PR DESCRIPTION
**Pull request by Emma-DJPR**
1. note.md created
2. The document was inserted into page methodology.R 
 Build and check showed one warning and 4 notes. Please check

### CODE REVIEW

File: R/note.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Who are we?
+
+**Strategy and Priority Projects (SPP)** supports DJPR with cross-portfolio priorities and policy development. SPP brings together advanced skills in public policy, strategy, program design, economics, modelling, data, evaluation and project management. The team uses these skills to provide a client-focused service offering across DJPR through a collaborative and flexible project-based approach to deliver first-rate public policy analysis and advice.
+
+**SPP's Data and Analytics** Unit works with diverse clients across DJPR to analyse data, build data products and facilitate data reporting, such as interactive dashboards, and improve data capabilities. We build economic models to answer a range of policy questions. If you have a policy question, we can work with you to answer it. Please follow the link for detail on what we do. (https://web4.economicdevelopment.vic.gov.au/ecodev/organisation/groups-and-branches/strategy-and-priority-projects/economic-and-data-analysis)
```
Comment (mrjoh3) [inline R/note.md:5]: the link in here is for the DJPR intranet, but the app will be public. Please replace the last 3 sentences with :

```
Please [email us](mailto:spp-data@ecodev.vic.gov.au?subject=DJPR%20Trade%20Dashboard) with any comments or feedback.
```

**Review by mrjoh3 — CHANGES_REQUESTED** [review]
need to change the last 3 sentences in the second paragraph. Please see the note in the PR

**Review by mrjoh3 — APPROVED** [review]

**Comment by mrjoh3** [comment]
@Emma-DJPR can you also the file to the methodology page. Once this is done I will merge

**Comment by Henry-DJPR** [comment]
edits superseded

**Pull request closed without merging**